config/migrations/20201224101445-publication-flow-ingetrokken.sparql

```diff
@@ -0,0 +1,27 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>
+PREFIX pubstatusid: <http://kanselarij.vo.data.gift/id/concept/publicatie-statussen/>
+PREFIX activity-type: <http://kanselarij.vo.data.gift/id/concept/activiteit-types/>
+PREFIX publish-type: <http://kanselarij.vo.data.gift/id/concept/publicatie-types/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX euvoc: <http://publications.europa.eu/ontology/euvoc#>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    pub:Publicatiestatus a ext:PublicClass .
+    pubstatusid:9b9b0b5e-45c8-11eb-b378-0242ac130002 a pub:Publicatiestatus ;
+      a skos:Concept ;
+      ext:priority 1 ;
```

Comment:
prioriteit 3?
die priors zijn er om ze te kunnen sorteren indien je een lijst hebt van statussen.
Prior 1 en 2 bestaan al in DB